certs/quasar-mongo-travis.pem

```diff
@@ -0,0 +1,54 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Yeah, I considered three options here
+ Use travis encrypted vars
+ Use `openssl` in build script 
+ Use nonencrypted pem file 

I originally preferred the second option, but then I realized that if I do all those steps locally there is no difference since the key/cert doesn't use anywhere else, and anyone can generate such key wtih such certificate in a minute or two.

Reply:
👍 I'm fine leaving this as-is